### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by wojtekkubiak**
Comment

### CODE REVIEW

File: new_file_in_dev.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+TO jest nowy plik w DEV
```
Comment (wojtekproton) [inline new_file_in_dev.txt:1]: Comment for line in new_file_in_dev

Comment (wojtekkubiak) [reply new_file_in_dev.txt:1]: ok

**Review by wojtekproton — APPROVED** [review]

**Review by wojtekproton — APPROVED** [review]

**Comment by wojtekproton** [comment]
Comment to Pull request from proton

**Comment by wojtekproton** [comment]
Closing

**Pull request merged**